feature/google/src/test/java/com/revenuecat/purchases/google/queryProductDetailsParamsExtensions.kt

```diff
@@ -0,0 +1,13 @@
+package com.revenuecat.purchases.google
+
+import com.android.billingclient.api.QueryProductDetailsParams
+
+internal val QueryProductDetailsParams.Product.productId
+    get() = this.zza()
+internal val QueryProductDetailsParams.Product.productType
+    get() = this.zzb()
+@Suppress("UNCHECKED_CAST")
+internal val QueryProductDetailsParams.productList: List<QueryProductDetailsParams.Product>
+    get() = this.zza() as List<QueryProductDetailsParams.Product>
+internal val QueryProductDetailsParams.productType
+    get() = this.zzb()
```

Comment:
These are just some handy extensions. These properties are obfuscated but public 🤷

Reply:
Yeah not great 🤦 but these should be useful for now 👍

Reply:
Does this need a new line here?